x-pack/plugins/observability/public/components/app/section/ux/index.tsx

```diff
@@ -7,25 +7,48 @@
 
 import { i18n } from '@kbn/i18n';
 import React from 'react';
+import type { AppDataType } from '../../../shared/exploratory_view/types';
 import { SectionContainer } from '..';
 import { getDataHandler } from '../../../../data_handler';
 import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
 import { useHasData } from '../../../../hooks/use_has_data';
 import { useDatePickerContext } from '../../../../hooks/use_date_picker_context';
+import { usePluginContext } from '../../../../hooks/use_plugin_context';
 import CoreVitals from '../../../shared/core_web_vitals';
 import { BucketSize } from '../../../../pages/overview';
+import { getExploratoryViewEmbeddable } from '../../../shared/exploratory_view/embeddable';
+import { AllSeries } from '../../../shared/exploratory_view/hooks/use_series_storage';
 
 interface Props {
   bucketSize: BucketSize;
 }
 
 export function UXSection({ bucketSize }: Props) {
   const { forceUpdate, hasDataMap } = useHasData();
+  const { core, plugins } = usePluginContext();
   const { relativeStart, relativeEnd, absoluteStart, absoluteEnd, lastUpdated } =
     useDatePickerContext();
   const uxHasDataResponse = hasDataMap.ux;
   const serviceName = uxHasDataResponse?.serviceName as string;
 
+  const ExploratoryViewEmbeddable = getExploratoryViewEmbeddable(core, plugins);
+
+  const seriesList: AllSeries = [
+    {
+      name: PAGE_LOAD_DISTRIBUTION_TITLE,
+      time: {
+        from: relativeStart,
+        to: relativeEnd,
+      },
+      reportDefinitions: {
+        'service.name': ['ALL_VALUES'],
+      },
+      breakdown: 'service.name',
+      dataType: 'ux' as AppDataType,
```

Comment:
This works for me without the `as AppDataType`. Maybe there was some other error before?

```suggestion
      dataType: 'ux',
```

Reply:
AppDataType probably should an enum.

Reply:
I'll do it in a different PR to make it easier to review.